database/models.py

```diff
@@ -1,10 +1,30 @@
 from .db import db
 import datetime
+from utils.utils import objectToJson
+
+class Author(db.Document):
+    name = db.StringField(required=True)
+    DOB = db.DateTimeField(required=True)
```

Comment:
keep column names in snake case whenever possible

Reply:
like `author_name`??

Reply:
Yes. Here, DOB can be replaced with dob, birth_date.